torch/backends/cudnn/__init__.py

```diff
@@ -3,11 +3,12 @@
 import warnings
 from contextlib import contextmanager
 from torch.backends import ContextProp, PropModule, __allow_nonbracketed_mutation
+from torch.types import _bool
```

Comment:
I don't think it should be necessary to use `_bool` here, as `torch.bool` isn't in scope here as `bool` (which is the reason for this workaround). Just use `bool` directly.